WeatherApp/WeatherApp/Feature/Components/WindWidgetView.swift

```diff
@@ -3,36 +3,34 @@ import SwiftUI
 struct WindWidgetView: View {
     var title: String
     var value: String
-    var deg: Double
+    var deg: Int
```

Comment:
degree

Reply:
WindWidget is changed and fixed in the previous PR